shared_tools/scripts/generate_standards_map.py

```diff
@@ -0,0 +1,652 @@
+'''
```

Comment:
Sorry for the unclear comment last time. It is suggested to move whole function "generate_standards_map()" to django command.